options.go

```diff
@@ -431,6 +431,8 @@ type PlainOpenOptions struct {
 	// DetectDotGit defines whether parent directories should be
 	// walked until a .git directory or file is found.
 	DetectDotGit bool
+	// Static means that the repository won't be modified while open.
+	Static bool
```

Comment:
so maybe readonly

Reply:
readonly?

Reply:
I thought about read only but what it really means is that it is not modified from the outside. I've added packfile and hash cache invalidation when new objects or packfiles are created. Mainly because I wanted to pass all tests in this mode.

Anyway, I agree that `Static` may not be the best name but could not find any other that reflected that is not modified from the outside.